src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,20 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long id);
+
+    List<Car> getAllByDriver(Long driverId);
+
```

Comment:
For the future, remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```